src/wavefront-obj-parser.js

```diff
@@ -2,47 +2,50 @@ module.exports = ParseWavefrontObj
 
 // Map .obj vertex info line names to our returned property names
 var vertexInfoNameMap = {v: 'vertex', vt: 'uv', vn: 'normal'}
-// The returned properties that we will populate
-var parsedProperties = ['normal', 'uv', 'vertex', 'normalIndex', 'uvIndex', 'vertexIndex']
+
 function ParseWavefrontObj (wavefrontString) {
+  'use strict'
```

Comment:
Mm yeah I was thinking that `standard` more or less forces our code to be strict, but it looks like in some cases explicitly going into strict mode can be faster. Alright LGTM